src/mlpack/core/optimizers/adam/optimisticadam_update.hpp

```diff
@@ -0,0 +1,144 @@
+/**
+ * @file optimisticadam_update.hpp
+ * @author Moksh Jain
+ *
+ * OptmisticAdam optimizer. Optimistic Adam is an an algorithm which uses
+ * Optimistic Mirror Descent with the Adam optimizer.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_ADAM_OPTIMISTICADAM_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_ADAM_OPTIMISTICADAM_UPDATE_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * OptimisticAdam is an optimizer which uses Optmistic Mirror Descent with
+ * the Adam Optimizer.
```

Comment:
The same line as above is repeated here. Ideally, it should be in one place only.

Reply:
Should I give more details here?

Reply:
If you can, take an example from `mlpack/methods/ann/layer/constant.hpp`.